GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/RO_BDB_Pioneer_NEW.cfg

```diff
@@ -0,0 +1,453 @@
+	%skinMaxTemp = 873.15
+	%crashTolerance = 12
+	%breakingForce = 250
+	%breakingTorque = 250
+}
+
+@PART[bluedog_Pioneer_TRD]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@mass = 0.001
+	@maxTemp = 773.15
+	@title = Pioneer 10/11 Jovian Trapped Radiation Detector
+	@manufacturer = University of California San Diego
+	%skinMaxTemp = 873.15
+	%crashTolerance = 12
+	%breakingForce = 250
+	%breakingTorque = 250
+}
+
+@PART[bluedog_Pioneer_CPD]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@mass = 0.001
+	@maxTemp = 773.15
+	@title = Pioneer 10/11/11 Charged Particle Instrument
+	@manufacturer = University of Chicago
+	%skinMaxTemp = 873.15
+	%crashTolerance = 12
+	%breakingForce = 250
+	%breakingTorque = 250
+}
+
+@PART[bluedog_Pioneer_InfraredRadiometer]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@mass = 0.001
+	@maxTemp = 773.15
+	@title = Pioneer 10/11 Infrared Radiometer
+	@manufacturer = California Institute of Technology
+	%skinMaxTemp = 873.15
+	%crashTolerance = 12
+	%breakingForce = 250
+	%breakingTorque = 250
+}
+
+@PART[bluedog_Pioneer_GeigerTube]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@mass = 0.001
+	@maxTemp = 773.15
+	@title = Pioneer 10/11 Geiger Tube Telescope
+	@manufacturer = University of Iowa
+	%skinMaxTemp = 873.15
+	%crashTolerance = 12
+	%breakingForce = 250
+	%breakingTorque = 250
+}
+
+@PART[bluedog_Pioneer_CRT]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@mass = 0.001
+	@maxTemp = 773.15
+	@title = Pioneer 10/11 Cosmic Ray Telescope
+	@manufacturer = Goddard Space Flight Center
+	%skinMaxTemp = 873.15
+	%crashTolerance = 12
+	%breakingForce = 250
+	%breakingTorque = 250
+}
+
+@PART[bluedog_Pioneer_UltravioletPhotometer]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@mass = 0.001
+	@maxTemp = 773.15
+	@title = Pioneer 10/11 Ultraviolet Photometer
+	@manufacturer = University of Southern California
+	%skinMaxTemp = 873.15
+	%crashTolerance = 12
+	%breakingForce = 250
+	%breakingTorque = 250
+}
+
+@PART[bluedog_Pioneer_PlasmaAnalyzer]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@mass = 0.0015
+	@maxTemp = 773.15
+	@title = Pioneer 10/11 Quadrispherical Solar Plasma Analyzer
+	@manufacturer = Ames Research Center
+	@description = This instrument is designed to detect solar wind particles originating from the Sun for the Pioneer 10/11 spacecraft. Intended to be mounted on top of the Pioneer 10/11 Experiment Payload Bay and peer through a hole in the Pioneer 10/11 High Gain Antenna's dish. 
+	%skinMaxTemp = 873.15
+	%crashTolerance = 12
+	%breakingForce = 250
+	%breakingTorque = 250
+}
+
+@PART[bluedog_Pioneer_CRT]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@mass = 0.001

```

Comment:
A 1kg telescope?

Reply:
Changed to 2 as it is in BDB original configs